resource-aggregate/events/deviceMetadataUpdated.go

```diff
@@ -69,11 +69,24 @@ func (d *DeviceMetadataUpdated) CheckInitialized() bool {
 		d.GetEventMetadata() != nil
 }
 
+func equal[T comparable](a, b []T) bool {
```

Comment:
slices.Equal (https://pkg.go.dev/slices#Equal)?

Reply:
for now ext package -> in 1.22 it is supported